src/App.js

```diff
@@ -1,12 +1,67 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import axios from "axios";
+import Pokemon from "./components /Pokemon";
+import pokemonlogo from "./assets/International_Pokémon_logo.svg.png"
+
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+
+    const [pokemonNames, setPokemonNames] = useState({});
+    const [offset, setOffset] = useState("https://pokeapi.co/api/v2/pokemon/")
+    const [loading, setLoading] = useState(false)
+    const [error, setError] = useState(false)
+
+    useEffect(() => {
+        async function getPokemons() {
+            setLoading(true)
+            setError(false)
+            try {
+                const {data} = await axios.get(offset);
+                console.log(data)
+                setPokemonNames(data);
+            } catch (e) {
+                console.log(e);
+                setError(true)
+            }
+            setLoading(false)
+        }
+
+        getPokemons()
+
+    }, [offset]);
+
+    return (
+        <div className="alles">
+            <div className="logo">
+                <img src={pokemonlogo} alt="pokemon logo"/>
+            </div>
+            <div className='buttons'>
+                <button
+                    type='button'
+                    disabled={!pokemonNames.previous}
+                    onClick={() => setOffset(pokemonNames.previous)}
+                >Vorige 20 pokemons
+                </button>
+
+                <button
+                    type='button'
+                    disabled={!pokemonNames.next}
+                    onClick={() => setOffset(pokemonNames.next)}
+                >Volgende 20 pokemons
+                </button>
+            </div>
+
+            <div className="pokemons">
+                {pokemonNames.results && pokemonNames.results.map((pokemon) => {
```

Comment:
Dan kun je deze ook nog iets versimpelen
```suggestion
                {pokemonNames?.results.map(pokemon => {
```